lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,44 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {Node[]} params
+ * @typedef {import('postcss-value-parser').Node} Node
+ * @returns {Array<string>}
+ */
+function listImportConditions(params) {
+	if (!params.length) return [];
+
+	/** @type {Array<String>} */
```

Comment:
Prefer `string` over `String`?

```suggestion
	/** @type {Array<string>} */
```

Reply:
Just trivial, please ignore it 😅